param_logger.go

```diff
@@ -27,13 +27,19 @@ type parameterLogger struct {
 	excluded []string
 }
 
-// ParameterLogger logs form and parameter values to the loggers
-func ParameterLogger(next buffalo.Handler) buffalo.Handler {
-	pl := parameterLogger{
-		excluded: ParameterExclusionList,
+func ParameterLoggerFiltering(customParams []string) buffalo.MiddlewareFunc {
```

Comment:
Rather than having another function, it could be nice if we have a single function `ParameterLogger(params ...string)` in the future release.